ios-base/DroidKaigi 2020/Extentions/Session+Ex.swift

```diff
@@ -3,7 +3,7 @@ import ios_combined
 
 extension Session {
     var timeRoomText: String {
-        "\(timeInMinutes)min / \(room.name.ja)"
+        "\(timeInMinutes)min / \(room.name.currentLangString)"
```

Comment:
I don't think you have time, so you don't need to change it,
You can use this method. 
`room.name.getByLang(lang: LangKt.defaultLang())`
https://github.com/DroidKaigi/conference-app-2020/blob/master/model/src/iosMain/kotlin/io/github/droidkaigi/confsched2020/model/Lang.kt#L7

Or you can create val currentLangString get() = getByLang(lang: LangKt.defaultLang()) in LocaledString 👀  
https://github.com/droidkaigi/conference-app-2020/blob/master/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2020/model/LocaledString.kt
However, I think that you can merge without changing. 😄

Reply:
I see. 
I think it is useful to create `val currentLangString get() = getByLang(lang: LangKt.defaultLang())` in `LocaledString.kt`, so I will try it! 👍

Reply:
Sorry like this!
val currentLangString:String get() = getByLang(defaultLang())

Reply:
I added in https://github.com/DroidKaigi/conference-app-2020/pull/754/commits/cdd127fb2c7fb099bc657757c1da046b27a8e2ac !
Please check this 🙏

Reply:
LGTM!!